### DISCUSSION TITLE
Mestastic t beam and BME 860

### DISCUSSION THREAD

**Discussion by lukeedwrads04** [cyberman54/ESP32-Paxcounter#1014] [Q&A]
Hi, I am very new to using meshtastic and am trying to connect the BME 680 to a t beam, I have been using the correct pins but for some reason wont show up the data in the environment sensor part of the app or on the LCD screen, had tried uploading code but now the board wont even connect via blutooth. I am getting a new one soon to try over but was wondering what the issue might be and how I could fix this? thanks

**Comment by cyberman54** [comment]
This is not the meshtastic project. You better should ask over there.
